data/templates/frr/bgp.frr.tmpl

```diff
@@ -203,6 +203,16 @@ router bgp {{ asn }}
 {%   endfor %}
 {% endif %}
  !
+{% if allow is defined %}
+{%   if allow.limit is defined %}
+ bgp listen limit {{ allow.limit }}
+{%   endif %}
+{%   for prefix in allow.range %}
```

Comment:
Can we please change this to `{%   for prefix, options in allow.range.items() %}` so that the `peer_group` key can be accessed by `options.peer_group` instead of `allow.range[prefix].peer_group`.

Reply:
Sure, I will make the changes as you want. Will be doing some testing to make sure the code works but I should be able to make all said required changes.